keps/sig-auth/541-external-credential-providers/README.md

```diff
@@ -198,8 +265,32 @@ Feature is already in Beta.
 
 #### Beta -> GA Graduation
 
-- 3 examples of real world usage
-- support for remote TLS handshakes (e.g. TPM/KMS-hosted keys)
```

Comment:
Dropped requirement.

Reply:
Exec plugins would potentially be the only way to use hardware or KMS-backed keys.
Do you think it's infeasible or not practical for k8s clients?

Reply:
I think the motivation for removing it is that we want this to GA, and don't have anyone working on TLS handshake offload. Removing it from GA blocking doesn't mean that we don't want to implement this at some later date.

Reply:
Part of me also thinks that using an environment variable as the "request" and standard output as the "response" may not be the best way to handle TLS handshakes.  Something like the Kube KMS proto API seems more appropriate.